versions.props

```diff
@@ -1,7 +1,7 @@
 org.slf4j:* = 1.7.25
 org.apache.avro:avro = 1.10.1
 org.apache.calcite:* = 1.10.0
-org.apache.flink:* = 1.12.5
+org.apache.flink:* = 1.14.3
```

Comment:
I don't think this is being used at all, but I updated it to the latest released Flink version.

Reply:
yeah. it does seem that we can remove this property

Reply:
There are a few that I think can be removed (eg the Spark dependencies).

One that I’m less sure about is Calcite

I can remove the Flink property in this PR or attempt to clean up several in a secondary PR (or both).